src/conf_mode/policy-route.py

```diff
@@ -123,6 +123,10 @@ def verify_rule(policy, name, rule_conf, ipv6):
                 for group in valid_groups:
                     if group in side_conf['group']:
                         group_name = side_conf['group'][group]
+
+                        if group_name and group_name[0] == '!':
```

Comment:
What about geoup_name.startswith()?

Reply:
the style mimics similar existing code in src/conf_mode/firewall.py:verify_rule()

Reply:
how many github "reviewers" does it take to change a lightbulb ?

it's trivial to fix, whether in existing code style or with startswith() or whatever.. but negated group support is important, and this bug breaks it so it should be fixed in whichever way or style you prefer !